external-api/api-tests/v1/party-db.api-test.ts

```diff
@@ -12,19 +12,82 @@
 
 import MockAdapter from 'axios-mock-adapter';
 import axios, { HttpStatusCode } from 'axios';
-import { MOCK_COMPANY_REGISTRATION_NUMBERS } from '@ukef/dtfs2-common';
+import { MOCK_COMPANY_REGISTRATION_NUMBERS, isAutomaticSalesforceCustomerCreationFeatureFlagEnabled } from '@ukef/dtfs2-common';
 import { app } from '../../src/createApp';
 import { api } from '../api';
 
 const { APIM_MDM_URL } = process.env;
 const { VALID, VALID_WITH_LETTERS } = MOCK_COMPANY_REGISTRATION_NUMBERS;
 const { get } = api(app);
+let axiosMock: MockAdapter;
+
+jest.mock('@ukef/dtfs2-common', () => ({
+  ...jest.requireActual('@ukef/dtfs2-common'),
+  isAutomaticSalesforceCustomerCreationFeatureFlagEnabled: jest.fn(),
+}));
+
+beforeEach(() => {
+  axiosMock = new MockAdapter(axios);
```

Comment:
No problem, just inherited the mocking from the existing (admittedly minimal) testing in this file. Is there a standard process in DTFS for setting up objects externally for testing?
If I want to create a customer in Salesforce to test that the GET returns 200 for a customer that exists, I'm assuming I should be setting up that customer within the test, and deleting it afterwards. There's not currently a way of deleting customers in SF via APIM, so that feels like a lot of extra code for this small change. Should I instead just set up a long-running customer manually in the QA/test environment to GET and just hope nobody comes along and deletes it? Wouldn't work in prod of course so then we'd have to xit this test in prod - all of which is to say it feels like maybe this wouldn't provide much value and mocking might be the right approach after all, what do you think @abhi-markan ?